src/Extensions/OrderExtension.php

```diff
@@ -228,7 +229,7 @@ public function formatDataForFlow()
             'ShipPostcode'         => $this->owner->ShippingAddressPostcode,
             'ShipState'            => $shippingAddressRegion,
             'ShipCountry'          => $shippingCountry,
-            'ShipNotes'            => $this->owner->ShippingAddressNotes,
+            'ShipNotes'            => str_replace(["\n","\r", '’', '>', '<'],['','', "'", '', ''], $this->owner->ShippingAddressNotes),
```

Comment:
I'm not sure about removing characters in this way.

Risky content should be escaped with `<![CDATA`.

see DBHTMLText::CDATA() method for example.

Reply:
Oh, we could write some unit tests for this right?